### ISSUE TITLE
Empty stylesheet for non-initial builds while watching with rollup

### ISSUE THREAD

**Issue by aktiur** [tivac/modular-css#288]
While configuring watching with Rollup and modular-css, I noticed that while the my style.css is correctly generated on first build, it is always empty for subsequent builds.

Here's my rollup.config.js:
```javascript
import resolve from 'rollup-plugin-node-resolve';
import css from 'modular-css-rollup';

export default {
  entry: 'src/main.js',
  format: 'iife',
  plugins: [
    css({
      css: 'dist/style.css'
    }),
    resolve()
  ],
  dest: 'dist/bundle.js',
  moduleName: 'mymodule'
};
```

Here's where I import a css file:
```javascript
import css from './dashboard.css';

function dashboard(root) {
  const app = root.append('div');

  app.attr('class', css.app)
    .text('Salut la compagnie !');
}

export default dashboard;
```

Here's the css file that is imported:
```css
.app {
    color: red;
}
```

And the resulting bundled `style.css` when first running `rollup -c -w`:
```css
/* src/components/dashboard.css */
.mc26720e23_app {
    color: red
}
/*# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uL3NyYy9jb21wb25lbnRzL2Rhc2hib2FyZC5jc3MiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBQUEsa0NBQUM7QUFBRDtJQUNJLFVBQVc7Q0FDZCIsImZpbGUiOiJzdHlsZS5jc3MiLCJzb3VyY2VzQ29udGVudCI6WyIuYXBwIHtcbiAgICBjb2xvcjogcmVkO1xufVxuIl19 */
```

And how it looks after changing any file (including the one that imports the CSS file).

```css

/*# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IiIsImZpbGUiOiJzdHlsZS5jc3MifQ== */
```

Here are my dev dependencies:
```javascript
  "devDependencies": {
    "eslint": "^3.19.0",
    "modular-css-rollup": "^4.1.7",
    "rollup": "^0.41.6",
    "rollup-plugin-node-resolve": "^3.0.0",
    "rollup-watch": "^3.2.2"
  },
```

I'm not sure what I did wrong!

**Comment by tivac** [comment]
You might not have done anything wrong! I don't think there is currently a test for usage w/ `rollup-watch`, so there's definitely a chance it's bugged. I'll take look 😅

**Comment by tivac** [comment]
Found that the test does exist but was a little broken, so fixed that. Seems to be working fine though.

Could you maybe provide a repo that illustrates the problem? The test runs against the `rollup-watch` API instead of using the CLI, so maybe that's it I dunno.

**Comment by aktiur** [comment]
Will try to create a minimal example when I get the time

**Comment by tivac** [comment]
Ok thanks, I really appreciate it! If I get some free time soon here I'll try to repro via the CLI.

**Comment by tivac** [comment]
@aktiur any further information you can provide would be great. I still haven't been able to repro this.

**Comment by aktiur** [comment]
Ok, I eventually found the time.

To reproduce :

Clone the following repo : https://github.com/aktiur/rollup-css-module-bug
```bash
> git clone git@github.com:aktiur/rollup-css-module-bug.git
```
Change directory and install node dependencies :
```bash
> cd rollup-css-module-bug
> npm i
```

Run rollup with watch option:
```bash
> node_modules/.bin/rollup -c -w
```

Verify that `dist/style.css` is not empty in another terminal:
```bash
> cat dist/style.css
/* src/main.css */
html {
    margin: 0;
    width: 100%;
    height: 100%
}
/*# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uL3NyYy9tYWluLmNzcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFBQSxrQkFBQztBQUFEO0lBQ0ksVUFBVTtJQUNWLFlBQVk7SUFDWixZQUFhO0NBQ2hCIiwiZmlsZSI6InN0eWxlLmNzcyIsInNvdXJjZXNDb250ZW50IjpbImh0bWwge1xuICAgIG1hcmdpbjogMDtcbiAgICB3aWR0aDogMTAwJTtcbiAgICBoZWlnaHQ6IDEwMCU7XG59XG4iXX0= */
```

Now add empty line to `src/main.js` in another terminal, and make sure the bundle has been recompiled.

Now verify that `dist/style.css` is empty, except for source mapping comment
```bash
> cat dist/style.css

/*# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IiIsImZpbGUiOiJzdHlsZS5jc3MifQ== */
```

**Comment by tivac** [comment]
Awesome! I'll try to stare at this real hard and come up with a solution soon. I'm sorry for the trouble 😓

**Comment by tivac** [comment]
@aktiur This looks fixed to me, probably by #301. I'll do a publish of that shortly, once that happens this should be fixed in `modular-css-rollup@5.1.1`.

Re-open if it still repros for you though please!
